types/lodash/lodash-tests.ts

```diff
@@ -3221,7 +3221,7 @@ namespace TestXor {
     }
 
     {
-        let result: _.LoDashExplicitArrayWrapper<TResult[]>;
+        let result: _.LoDashExplicitArrayWrapper<Array<TResult | undefined>>;
```

Comment:
It's not the right way to test types because it's possible to assign anything compatible to this variable, not only the exact specified type from the annotation. E.g. TypeScript expectedly allows assigning `[1,2,3]` to `var foo: Array<number | undefined>`.
Please [use `$ExpectType` instead](https://github.com/DefinitelyTyped/DefinitelyTyped#lint).

Reply:
Good suggestion. Updated!